src/appengine/handlers/cron/project_setup.py

```diff
@@ -611,7 +609,7 @@ def _get_build_bucket(self, engine, architecture):
 
   def _deployment_bucket_name(self):
     """Deployment bucket name."""
-    return '{project}-deployment'.format(project=utils.get_application_id())
+    return f'{utils.get_application_id()}-deployment'
```

Comment:
There's not really a problem here, but I wonder if it's better to limit the complexity of the python code in f-strings.

Reply:
Yeah, I don't think there is any problem with the original code here (or all other code in Commit [format and lint](https://github.com/google/clusterfuzz/pull/2578/commits/eb09218715f6fc177545bfbfde7e84802f818d3c)).  
I edited them just to pass the `lint` CI check (`python bulter.py lint`).
Sorry about the confusion.